r/tests/testthat/test-dplyr-group-by.R

```diff
@@ -62,9 +62,9 @@ test_that("ungroup", {
       select(int, chr) %>%
       ungroup() %>%
       filter(int > 5) %>%
+      collect() %>%
       summarize(min_int = min(int)),
```

Comment:
```suggestion
      # TODO: ARROW-13550 - remove summarize() from here (expect_dplyr_equal will check attributes)
      collect() %>%
      summarize(min_int = min(int)),
```

Reply:
@thisisnic could you clarify what you mean by this? Thanks

Reply:
Yeah, sorry, that was a bit vague! I'm assuming here that `summarize()` is being called at the end of here to make sure that the `group_by()` has happened successfully - currently `expect_dplyr_equal()` uses `expect_equivalent()` and so ungrouped and grouped tibbles "look" the same to it.

In that PR, `expect_dplyr_equal()` has been updated to use `expect_equal()` which will check for attributes, and so comparing a grouped and ungrouped tibble will result in a test failure, meaning that the `summarize()` here is no longer needed (if the only purpose for it is checking that `group_by()` has worked.

Reply:
Ah, makes sense, thanks!